__tests__/get-routes.test.ts

```diff
@@ -11,10 +11,10 @@ import {
 import { requestTestToken } from "../__test-utils/req-test-token";
 // byid ✓
 // watered ✓
-// all ✓
+// all ✓ DEPRECATED
```

Comment:
I would go so far to even remove the comment lines altogether.

Reply:
Jupp. Removed.